### ISSUE TITLE
Unable to compile?

### ISSUE THREAD

**Issue by Kinny092** [elddy/NimScan#14]
Tried gcc test.c -L. -1:NimScanToC.a -w -o NimScan.exe 

came up with heap of errors

is this the right command?

**Comment by elddy** [comment]
Updated the repo.
Can you try again?
Also, you used -1 and not -l

**Comment by Kinny092** [comment]
Yeah I actually use -l and it showed a long error. -1 does not work at all. not sure why i had that inaccurate description.

My error is here: https://paste.rs/S5D

**Comment by elddy** [comment]
Have you tried to recompile after the update?
What are your GCC and Nim versions?
